AutoNode/__init__.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sys
+import json
+import warnings
+
+from pyhmy import (
+    cli,
+    Typgpy
+)
+
+from .common import (
+    harmony_dir,
+    node_dir,
+    node_sh_log_dir,
+    bls_key_dir,
+    saved_validator_path,
+    imported_bls_pass_file_dir,
+    imported_wallet_pass_file_dir,
+    cli_bin_path,
+    saved_node_path,
+    validator_config,
+    node_config
+)
+
+
+if sys.version_info.major < 3:
+    warnings.simplefilter("always", DeprecationWarning)
+    warnings.warn(
+        DeprecationWarning(
+            "`AutoNode` does not support Python 2. Please use Python 3."
+        )
+    )
+    warnings.resetwarnings()
+
+if sys.platform.startswith('win32') or sys.platform.startswith('cygwin'):
+    warnings.simplefilter("always", ImportWarning)
+    warnings.warn(
+        ImportWarning(
+            "`AutoNode` does not work on Windows or Cygwin."
+        )
+    )
+    warnings.resetwarnings()
```

Comment:
same, if it doesn't work, then exit. Suggest people use WSL  ,should work under that

Reply:
Added exit and suggestion